.github/workflows/it.yaml

```diff
@@ -4,7 +4,7 @@ on:
   pull_request:
   push:
     branches:
-      - master
+      - build-docker-images
```

Comment:
Will update when CI passes

Reply:
Thanks! I will fix the test after this [PR](https://github.com/alephium/wormhole-fork/pull/37) is merged.